### ISSUE TITLE
Warpinator menu icon missing after this morning's update

### ISSUE THREAD

**Issue by mfreeman72** [linuxmint/warpinator#55]
Warpinator was updated this morning, and now its menu icon is missing. I restarted by computer, and it was still missing. I opened the menu editor to manually assign an icon, and entering "warpinator" in the icon selector's search bar reveals no icon exists. The icon existed on my system before the update.

**Comment by mtwebster** [comment]
https://github.com/linuxmint/warpinator/issues/54

You may need to install that package.

Is there a 'warpinator.desktop' in /usr/share/applications?  If so what are its contents?

**Comment by mfreeman72** [comment]
Yes, there is a warpinator.desktop file. I don't think that's the issue. I can see the entry in the menu, and I can run it. It just doesn't have an icon. I will attach a screenshot of my menu to illustrate what I mean:

![Missing Warpinator Icon](https://user-images.githubusercontent.com/7842664/90282711-b16aa780-de23-11ea-9df5-f5011128029d.png)

My /usr/share/applications/warpinator.desktop contents are:

[Desktop Entry]
Type=Application
Name=Warpinator
Comment[ca]=Envia i rep arxius a través de la xarxa
Comment[cs]=Posílání a přijímání souborů po síti
Comment[da]=Send og modtag filer over netværket
Comment[de]=Dateien über das Netzwerk senden und empfangen
Comment[en_GB]=Send and Receive Files across the Network
Comment[eo]=Sendi kaj ricevi dosierojn tra la reto
Comment[es]=Enviar y recibir archivos a través de la red
Comment[et]=Failid saatmine ja vastuvõtmine võrgu kaudu
Comment[eu]=Bidali eta jaso fitxategiak sarean zehar
Comment[fi]=Lähetä ja vastaanota tiedostoja verkossa
Comment[fr]=Envoyer et recevoir des fichiers sur le réseau
Comment[he]=שליחת וקבלת קבצים על פני הרשת
Comment[hi]=नेटवर्क द्वारा फाइलों का आदान-प्रदान करें
Comment[hr]=Šaljite i primajte datoteke putem mreže
Comment[hu]=Fájlok küldése és fogadása a hálózaton
Comment[id]=Kirim dan Terima berkas melalui Jaringan
Comment[is]=Senda og taka við skrám um netið
Comment[it]=Invia e Ricevi Files sulla Rete
Comment[ja]=ネットワーク経由でファイルを送受信します
Comment[kab]=Azen rnu ṭṭef-d ifuylas uzeṭṭa
Comment[lt]=Siųsti ir gauti failus per tinklą
Comment[nl]=Bestanden verzenden en ontvangen via het netwerk
Comment[pl]=Wysyłaj i odbieraj pliki w swojej sieci
Comment[pt]=Enviar e receber ficheiros através da rede
Comment[pt_BR]=Envie e Receba Arquivos através da Rede
Comment[ro]=Trimite și Primește Fișiere peste Rețea
Comment[ru]=Отправка и получение файлов по сети
Comment[sr@latin]=Šaljite i primajte datoteke preko mreže
Comment[sv]=Skicka och ta emot filer via nätverket
Comment[tr]=Ağ Üzerinden Dosya Gönderin ve Alın
Comment[uk]=Відправка й отримання файлів мережею
Comment=Send and Receive Files across the Network
Icon=warpinator
Exec=warpinator
Categories=GNOME;GTK;Utility;

**Comment by mtwebster** [comment]
Ah I think I see the problem - are you using Mint-Y icon theme? If not can you try it to see if the icon shows up?

I've been working on packaging this for flatpak, and I've had to do a bit of renaming of things to follow the conventions there.

**Comment by mfreeman72** [comment]
Yep, that's the problem. I am not using the Mint-Y theme (it's my least favorite of all the themes). I'm using the Mint-X theme currently. The icon was there before the warpinator update. I'm assuming it just didn't get linked into the Mint-X theme for some reason (or however that works)?

**Comment by mtwebster** [comment]
The original icon name is in Mint-Y still (warpinator).

The new icon names packaged with warp (which would be used in any other theme lacking their own) are now named 'org.x.Warpinator'.  The desktop file is still wanting to use the old name.

**Comment by mfreeman72** [comment]
Yes, that fixed the issue.

**Comment by mfreeman72** [comment]
I hope this doesn't mean that the Mint-X style of icon theming isn't going to be supported going forward. The Warpinator icon style definitely looks Mint-Y-ish, I've noticed.

**Comment by mfreeman72** [comment]
Anyway, thanks for your help on this!

**Comment by Harry-W-Haines-III** [comment]
I am using LMDE 4 and accepted the 1.0.7+debbie package.  I was able to restore the icon using the fix noted in above.  It still won't start.  I get this message trying to start it from the command prompt.
harry@radio-desk:~$ warpinator
No module named 'packaging'

harry@radio-desk:~$ apt list warpinator
Listing... Done
warpinator/debbie,debbie,now 1.0.7+debbie all [installed]

Am I missing something here?

**Comment by mtwebster** [comment]
Yes, see the issue I linked above, you need to install 'python3-packaging'.  It's a new dependency and we screwed up.  It's fixed, we just need to issue a new release.

**Comment by French77** [comment]
### Go to `usr/share/applications`
#### Open application as administrator (right click followed by password)
#### Open warpinator with your text editor,and change Icon to `org.x.Warpinator` (and save as)
![Screenshot](https://i.imgur.com/FQ1nYOx.png.png"Screenshot")]
Icon is back in menu
![menu warpinator](https://user-images.githubusercontent.com/3099270/90341567-e3853080-e000-11ea-9d02-239eb2c394aa.png)

**Comment by Harry-W-Haines-III** [comment]
Why not have warpinator_1.0.6+debbie_all.deb left in the repository so users who wish to do so can back rev to a working version?  I have looked everywhere and cannot find it.
